pkg/operator/ceph/object/admin.go

```diff
@@ -221,7 +221,7 @@ func RunAdminCommandNoMultisite(c *Context, expectJSON bool, args ...string) (st
 	}
 
 	if err != nil {
-		return fmt.Sprintf("%s. %s", output, stderr), err
+		return output, errors.Wrapf(err, "%v", stderr)
```

Comment:
```suggestion
		return output, errors.Wrapf(err, "%v", stderr)
```

@BlaineEXE @travisn this is what I was trying to point out yesterday. I'll try to repro the user issue and see if this actually fixes. But I do think this is some bug.

Reply:
The existing code seems correct to me. Seems like the issue would be that the caller is either not logging the err or the output. Since all the output, stderr, and err are being returned in some form, the burden would be on the caller to log them or handle them appropriately.